SPECS/iscsi-initiator-utils/iscsi-initiator-utils.spec

```diff
@@ -47,6 +47,8 @@ BuildRequires: flex bison doxygen kmod-devel systemd-units
 BuildRequires: autoconf automake libtool libmount-devel openssl-devel
 BuildRequires: isns-utils-devel
 BuildRequires: systemd-devel
+BuildRequires: systemd-rpm-macros
```

Comment:
is adding rpm-macros mistake?  it solves the current `package systemd-255-4.azl3.x86_64 requires (systemd-rpm-macros = 255-4.azl3 if rpm-build), but none of the providers can be installed` issue, but it really seems like the issue should be solved elsewhere.

Reply:
removed